test/ref/continuous_test.lst

```diff
@@ -66,6 +66,7 @@ GeneralizedExtremeValue(1.0, 1.0, -1.0)
 GeneralizedExtremeValue(-1.0, 0.5, 0.6)
 
 GeneralizedPareto()
+GeneralizedPareto(1.0)
```

Comment:
Could we also add something like
```suggestion
GeneralizedPareto(1.0)
GeneralizedPareto(2.0)
```
? That seems a bit safer since `GeneralizedPareto(1.0)` would pass even if e.g. it defaults to `GeneralizedPareto()` or if the argument would set the scale instead of the shape.
?